deployments/ansible/roles/collector/tasks/linux_install.yml

```diff
@@ -37,6 +37,29 @@
     mode: 0600
   notify: "restart splunk-otel-collector"
 
+- name: Read the default config
+  slurp:
+    path: "{{splunk_otel_collector_config}}"
+  register: defaultconfig
+  when: splunk_config_override != ''
+
+- name: Extract the default config
+  set_fact:
+    updated_config: "{{ defaultconfig['content'] | b64decode | from_yaml }}"
+  when: splunk_config_override != ''
+
+- name: Update the default config
+  set_fact:
```

Comment:
Please use full names for modules as established practice in this role

Reply:
Ok, Sure.